mplc/docs/documentation.md

```diff
@@ -298,7 +298,77 @@ All methods available are:
+
+  - [done, short name "TMCS"] **[Truncated Monte-Carlo Shapley](https://arxiv.org/pdf/1904.02868.pdf) approximation**:  
+  The idea of Truncated Monte-Carlo is that, for a large coalition, the increments of a player are usually small, therefore we can consider their value is null instead of spending computational power to compute it. This reduce the number of times we have to fit a model, but adds a small bias in the estimation.
+
+  - [done, short name "ITMCS"] **Interpolated Truncated Monte-Carlo Shapley**:  
+  This method is an attempt to reduce the bias of the Truncated Monte-Carlo Shapley method. Here we do not consider that the value of an increment of a large coalition is null, but we do a linear interpolation to better approximate its value.
+
+- **Importance sampling methods**:
+
+  Importance sampling is a method to reduce the number of sampled increments in the Monte-Carlo method while keeping the same accuracy. It consists in sampling the increments according to non-uniform distribution, giving more chance for big increment than for small increment to be sampled. The bias induced by altering the sampling distribution is canceled by properly weighting each sample: if an increment is sampled with *X* times more chances, then we weight it by *1/X*. Note that this require to know the value of increment before computing them, so in practice we try to guess the value of the increment. We inflate, resp. deflate, the probability of sampling an increment if we guess that its value is high, resp. small. We designed three ways to guess the value of increments, which lead to three different importance sampling methods:
+
+  - [done, short name "IS_lin_S"] **Linear importance sampling**
+  - [done, short name "IS_reg_S"] **Regression importance sampling**
+  - [done, short name "AIS_Kriging_S"] **Adaptative Kriging importance sampling**
+
+- **[Stratified Monte Carlo Shapley](https://arxiv.org/pdf/1904.02868.pdf)**:
+
+  "Stratification and with proper allocation" is another method to reduce the number of sampled increments in the Monte-Carlo method while keeping the same accuracy. There are two ideas behind this method:
+
+  1. The Shapley value is a mean of means taken on strata of increments. A strata of increments corresponds to all the increments of coalitions with the same number of players. We can estimate the means on each strata independently rather than the whole mean, this improves the accuracy and reduces the number of increments to sample.
+  1. We can allocate a different amount of sampled increment to each mean of a strata. If we allocate more sample to the stratas where the increments value varies more, we can reduce the accuracy even more.
```

Comment:
2 instead of 1.

Reply:
In markdown, you can use only `1.` and it will be auto incremented when rendered, but yes, it is troubling...